app/globals.py

```diff
@@ -24,7 +24,10 @@ def get_metadata(user):
         return None
 
     questionnaire_store = get_questionnaire_store(user.user_id, user.user_ik)
-    return questionnaire_store.metadata
+    if questionnaire_store.metadata:
```

Comment:
If this is false then `questionnaire_store.metadata` would be None? therefore can you not just return `questionnaire_store.metadata` as it was before?

Reply:
It will always exist but will be empty after submission. This logic stops it returning an empty dictionary.

Reply:
Yeh if this is the only place we set this to `None` then the other code should just handle the empty dict.